src/server/redis_connection.cc

```diff
@@ -172,6 +172,15 @@ std::string Connection::ArrayOfSet(const std::vector<std::string> &elems) const
   return result;
 }
 
+std::string Connection::MapOfBulkStrings(const std::vector<std::string> &elems) const {
+  std::string result;
```

Comment:
Could you add an assertion here? e.g. `CHECK(elems.size() % 2 == 0)`

Reply:
@PragmaTwice Sure, done.